cmd/jujud/agent/machine/manifolds.go

```diff
@@ -693,10 +693,10 @@ func commonManifolds(config ManifoldsConfig) dependency.Manifolds {
 		})),
 
 		dbAccessorName: ifController(dbaccessor.Manifold(dbaccessor.ManifoldConfig{
-			AgentName: agentName,
-			Clock:     config.Clock,
```

Comment:
Is this a mistake? We've still got a clock member and both mocks and logic for it...